docs/install.md

```diff
@@ -1,809 +1,131 @@
-   will create a cluster running locally, with RBAC enabled and your user granted
-   the `cluster-admin` role.
-
-1. If you want to support high availability usecases, install a [Metrics Server](https://github.com/kubernetes-sigs/metrics-server) on your cluster.
-
-1. Choose the version of Tekton Pipelines you want to install. You have the following options:
-
-   * **[Official](https://github.com/tektoncd/pipeline/releases)** - install this unless you have
-     a specific reason to go for a different release.
-   * **[Nightly](../tekton/README.md#nightly-releases)** - may contain bugs,
-     install at your own risk. Nightlies live at `gcr.io/tekton-nightly`.
-   * **[`HEAD`]** - this is the bleeding edge. It contains unreleased code that may result
-     in unpredictable behavior. To get started, see the [development guide](https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md) instead of this page.
-
-1. Grant `cluster-admin` permissions to the current user.
-
-   See [Role-based access control](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#prerequisites_for_using_role-based_access_control)
-   for more information.
-
-## Installing Tekton Pipelines on Kubernetes
+{{% comment %}}
+To view the full contents of this page, go to the 
+<a href="http://tekton.dev/docs/installation/pipelines/">Tekton website</a>.
+{{% /comment %}}
 
-To install Tekton Pipelines on a Kubernetes cluster:
+{{% pageinfo %}}
+{{% readfile "/vendor/disclaimer.md" %}}
+{{% /pageinfo %}}
 
-1. Run the following command to install Tekton Pipelines and its dependencies:
+This guide explains how to install Tekton Pipelines.
 
-   ```bash
-   kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
-   ```
+## Prerequisites
 
-   Or, for the nightly release, use:
+-   A [Kubernetes cluster][k8s] running version 1.22 or later.
+-   [Kubectl][].
+-   Grant `cluster-admin` privileges to the current user. See the [Kubernetes
+    role-based access control (RBAC) docs][rbac] for more information.
+-   (Optional) Install a [Metrics Server][metrics] if you need support for high
+    availability use cases.
 
-   ```bash
-   kubectl apply --filename https://storage.googleapis.com/tekton-releases-nightly/pipeline/latest/release.yaml
-   ```
+See the [local installation guide][local-install] if you want to test Tekton on
+your computer.
 
-   You can install a specific release using `previous/$VERSION_NUMBER`. For example:
+[k8s]: https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/
+[kubectl]: https://kubernetes.io/docs/tasks/tools/#kubectl
+[rbac]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+[metrics]: https://github.com/kubernetes-sigs/metrics-server
+[local-install]: https://tekton.dev/docs/installation/local-installation/
```

Comment:
NIT: perhaps we could have all the links at the bottom of the doc

Reply:
I moved them to the bottom.